samples/Mvc.Formatters/Mvc.Formatters/Views/Home/Index.cshtml

```diff
@@ -0,0 +1,36 @@
+﻿@{
+    ViewData["Title"] = "Posting data in text/plain format";
+}
+<h2>Posting data in text/plain format</h2>
+<div>
+    <label>Sample text:</label>
+    <input type="text" id="sampleText" />
+    <button id="submitText" onclick="javascript: submitData()">Submit</button>
+    <div id="divStatus" style="margin-top:20px">
+        <label style="font-weight:bold">Echoed value: </label>
+        <label id="lblStatus"></label>
+    </div>
+</div>
+@section Scripts{
+    <script src="https://code.jquery.com/jquery-3.1.0.min.js"></script>
+    <script type="text/javascript">
```

Comment:
Yeah I actually did try that with `enctype=text/plain` in the form element and was expecting only the input field value to be sent over the wire but i see that it still goes as key-value pair. I didn't want this to be shown in the sample to avoid confusing with form url encoded format which typically is considered a key-value format type data.